apmpackage/apm/data_stream/internal_metrics/fields/lambda_metrics.yml

```diff
@@ -0,0 +1,20 @@
+- name: aws.lambda.metrics.billed_duration
```

Comment:
I agree with a possible change to `faas.billed_duration` - however, we might want to keep a naming element allowing us to differentiate these metrics from the `faas` fields introduced in the model.